artman/pipelines/grpc_generation.py

```diff
@@ -106,19 +111,23 @@ class _GoGrpcTaskFactory(GrpcTaskFactoryBase):
     """
 
     def _get_grpc_codegen_tasks(self, **kwargs):
-        return [protoc_tasks.ProtoAndGrpcCodeGenTask,
-                protoc_tasks.GoExtractImportBaseTask,
-                protoc_tasks.GoLangUpdateImportsTask]
+        return [
+            protoc_tasks.ProtoAndGrpcCodeGenTask,
+            protoc_tasks.GoExtractImportBaseTask,
+            protoc_tasks.GoLangUpdateImportsTask,
+        ]
 
     def get_validate_kwargs(self):
-        return ['gapic_api_yaml', 'final_repo_dir'] + code_gen.COMMON_REQUIRED
+        return ['gapic_api_yaml', 'gapic_code_dir'] + code_gen.COMMON_REQUIRED
```

Comment:
Actually, I think I am wrong about this. I was thinking of `gapic_subpath` and `grpc_subpath`.
I will see if this is readily rename-able.